src/styles/transformers/rtl.js

```diff
@@ -0,0 +1,90 @@
+import merge from '../../merge';
+import warning from 'warning';
+
+const reTranslate = /((^|\s)translate(3d|X)?\()(\-?[\d]+)/;
+const reSkew = /((^|\s)skew(x|y)?\()\s*(\-?[\d]+)(deg|rad|grad)(,\s*(\-?[\d]+)(deg|rad|grad))?/;
+
+/**
+ * This function ensures that `style` supports both ltr and rtl directions by
+ * checking `styleConstants` in `muiTheme` and replacing attribute keys if
+ * necessary.
+ */
+export default function rtl(muiTheme, style) {
```

Comment:
Oh right, I didn't like the idea that theme could be modified (and changes passed) through the transform chain. Though I saw that redux middleware does something to the sort this (where getTheme is getState):

```js
export default ({getTheme}) => style => {
// getTheme is a function that returns the theme
};